e2e/challenge-description.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { test, expect, type Page } from '@playwright/test';
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto(
+    '/learn/foundational-c-sharp-with-microsoft/write-your-first-code-using-c-sharp/trophy-write-your-first-code-using-c-sharp'
+  );
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
```

Comment:
```suggestion
test.beforeEach(async ({ page }) => {
  await page.goto(
    '/learn/foundational-c-sharp-with-microsoft/write-your-first-code-using-c-sharp/trophy-write-your-first-code-using-c-sharp'
  );
});
```

We should ideally run tests in isolation.